### PR TITLE
Added github action

### PR DESCRIPTION
**Pull request by e-minguez**
A first try to have a github action. See the results of a successful run here https://github.com/e-minguez/test-gha/runs/7132759775?check_suite_focus=true

### CODE REVIEW

**Comment by manuelbcd** [comment]
LGTM
I'm adding @guipal

**Comment by e-minguez** [comment]
@airadier would you mind to take a look? Thanks!

File: github/scan.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      uses: docker/build-push-action@v3
+      with:
+        context: github/
+        tags: ${{ env.REGISTRY_HOST }}/${{ secrets.REGISTRY_USER }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}
+        load: true
+
+    - name: Download sysdig-cli-scanner if needed
+      run:  |
+        curl -sLO https://download.sysdig.com/scanning/sysdig-cli-scanner/latest_version.txt
+        mkdir -p /tmp/{db,cache}
```
Comment (airadier) [inline github/scan.yaml:31]: Do we need to keep the latest_version.txt in /tmp/cache? I kind of understand using /tmp/db for the vulnerability mini DB and using the github action cache feature to persist between executions, but what about the latest_version.txt? I think the installation would like simpler using the one-liner provided in the docs.

**Review by airadier — APPROVED** [review]

File: github/scan.yaml (hunk 1 above)
Comment (e-minguez) [reply github/scan.yaml:31]: My plan is to create a follow up PR to leverage cache for both the databases and the binary itself, so if the binary has been already downloaded and is the same version it doesn't need to be downloaded again.

**Comment by e-minguez** [comment]
Is there anything else needed for this to be merged? Thanks

**Comment by airadier** [comment]
> Is there anything else needed for this to be merged? Thanks

> Is there anything else needed for this to be merged? Thanks

Not really, but for the sake of simplicity I would suggest keeping the example minimal or complete the pending items (enable cache). What do you think?

**Comment by e-minguez** [comment]
I've added support for the github actions cache. See a successful run here https://github.com/e-minguez/test-gha/runs/7132759775?check_suite_focus=true

**Pull request merged**